src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,64 @@
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+// const DrcHelpers = require('../utils/helpers.js');
+// const ENVIRONMENTS = require('site/constants/environments');
+// const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
+const inputs = require('./utils/inputs.json');
+const DrcE2eHelpers = require('./drc-helpers.js');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  // Navigate to beta-page.
+  client
+    .openUrl(`${E2eHelpers.baseUrl}/disability/about-disability-ratings-beta/`)
+    .waitForElementVisible('body', Timeouts.normal)
+    .assert.title('How VA Assigns Disability Ratings | Veterans Affairs');
```

Comment:
We've added the document title into E2E in the past, but it's usually led to a headache because the content team changes these values periodically. I think we can do without this assertion, just in case it's changed right before we're told to launch 😛

Reply:
Replace page-title assertion w/ waitForElement to check Calculator’s component-div.